stripe/src/main/java/com/stripe/android/util/StripeNetworkUtils.java

```diff
@@ -83,4 +118,35 @@ static void removeNullParams(Map<String, Object> mapToEdit) {
         }
     }
 
+    @SuppressWarnings("HardwareIds")
```

Comment:
Ignored warning is because Google doesn't want you to use android id for ad tracking. We aren't using it for that.